### PR TITLE
Added styled Form validation using Gandalf-Validation.

### PR DESCRIPTION
**Pull request by ringong95**
Added form validation using Gandalf.  Closes: #8

### CODE REVIEW

File: js/scenes/Hello/HelloContainer.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React, { Component, PropTypes } from 'react';
```
Comment (scottfuoco) [inline js/scenes/Hello/HelloContainer.js]: Hello Component is no longer needed

**Review by scottfuoco — CHANGES_REQUESTED** [review]
Everything looks good, just need remove the unused Hello Component.

**Review by BlakeTurner — CHANGES_REQUESTED** [review]
Good start.

In the Generic form you've laid out a pattern. This is good. However, code that we don't use has no place in the codebase. If you're going to put that anywhere, put it in the Wiki with a description of _how_ to implement forms.

Otherwise, it looks like the styles for login and signup are pretty much the same. Would it make more sense to create a FormStyles file and just import that in to both components?

**Review by BlakeTurner — APPROVED** [review]

**Review by scottfuoco — APPROVED** [review]
Good work!

**Pull request merged**